src/devolucion-tributaria/devolucion-tributaria.controller.ts

```diff
@@ -0,0 +1,41 @@
+import { Query,Controller, Res, Post, Get, Put, Delete, Param, Body, HttpStatus, NotFoundException} from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { DevolucionTributariaDto } from './dto/devolucion-tributaria.dto'
+import { DevolucionTributariaService } from './devolucion-tributaria.service';
+import { FilterDto } from '../shared/dto/filter.dto';
+
+@ApiTags('devolucion-tributaria')
+@Controller('devolucion-tributaria')
+export class DevolucionTributariaController {
+
+    constructor(private readonly devolucionTributariaService: DevolucionTributariaService) { }
+    @Post()
+    post(@Body() devolucionTributariaDto: DevolucionTributariaDto) {
+        return this.devolucionTributariaService.post(devolucionTributariaDto);
+    }
+    @Get()
+    get(@Query() filterDto: FilterDto) {
+        return this.devolucionTributariaService.getAll(filterDto)
+    }
+
+    @Get('/:id')
+    async getById(@Res() res, @Param('id') id: string) {
+        const devolucionTributaria = await this.devolucionTributariaService.getById(id);
+        if (!devolucionTributaria) throw new NotFoundException("not found resource");
```

Comment:
El `NotFoundException("not found resource")` cómo se refleja al Body de la respuesta? O sea, cómo queda el cuerpo de la respuesta cuando aplica el `throw`?

Reply:
![image](https://user-images.githubusercontent.com/38676533/169844125-1c6c0f0a-6aa4-46bf-9cc4-8d3229e33338.png)